### ISSUE TITLE
Unable to use cvsubentry

### ISSUE THREAD

**Issue by kedmond** [posquit0/Awesome-CV#29]
Thank you for the great template.

I've been having trouble using the \cvsubentry commnad.

```
\begin{cvsubentries}
  \cvsubentry
	{Position}
	{title}
	{today -- tomorrow}
	{description}
\end{cvsubentries}
```

I'm not sure what I'm doing wrong here. I receive an error stating that 
> ! Extra alignment tab has been changed to \cr.
><recently read> \endtemplate
>l.26 ^^I {description}
>You have given more \span or & marks than there were
>in the preamble to the \halign or \valign now in progress.
>So I'll assume that you meant to type \cr instead.

I've tried many different things, such as placing the sub-entry environment within the entry environment, etc.  Can you provide a .tex file with an example of its usage? Thanks again.

**Comment by posquit0** [comment]
Fixed this problem #28 
Thanks!

**Comment by mpeterson** [comment]
I happen to continue to have this problem, cvsubentry is not working correctly when all parameters are provided.

**Comment by johannesbottcher** [comment]
@mpeterson Please post a compilable example to let us reproduce the issue. Did you download the updated template?

**Comment by mpeterson** [comment]
@johannesbottcher Sorry for the delay. I cloned the Git repository and it was updated, so yes.

Using the example that sits on the example folder of the repository if you have this "complex" entry it won't work. 

`  \cventry
    {Title} % Job title
    {Org} % Organization
    {Location} % Location
    {Feb. 2009 - May. 2012} % Date(s)
    {
      \begin{cvsubentries}
        \cvsubentry
          {Sub Title 1}
          {Sub Org 1}
          {Dec. 2009 - May. 2012}
          {
          \begin{cvitems} % Description(s) of tasks/responsibilities
            \item {item 1.}
            \item {item 2.}
          \end{cvitems}
          }
        \cvsubentry
          {Sub Title 2}
          {Sub Org 2}
          {Feb. 2009 - Nov. 2009}
          {
          \begin{cvitems}
            \item {Item 1}
            \item {Item 2}
          \end{cvitems}
          }
      \end{cvsubentries}
}`

But it wouldn't work either with this simple entry that was part of the examples that was provided (once you add a title and a description:

`  \cventry
    {Freelance Penetration Tester} % Job title
    {SAMSUNG Electronics} % Organization                                                                                                                                               {S.Korea} % Location
    {Sep. 2013, Mar. 2011 - Oct. 2011} % Date(s)
    {
      \begin{cvitems} % Description(s) of tasks/responsibilities
        \item {Conducted penetration testing on SAMSUNG KNOX, which is solution for enterprise mobile security.}
        \item {Conducted penetration testing on SAMSUNG Smart TV.}
      \end{cvitems}
      \begin{cvsubentries}
          \cvsubentry{added title}{KNOX(Solution for Enterprise Mobile Security) Penetration Testing}{Sep. 2013}{added description}
          \cvsubentry{added title}{Smart TV Penetration Testing}{Mar. 2011 - Oct. 2011}{added description}
      \end{cvsubentries}
    }`

**Comment by johannesbottcher** [comment]
Please post a *compilable* example.

**Comment by johannesbottcher** [comment]
That issue was never fixed, i'll take a look at the mess.

**Comment by johannesbottcher** [comment]
@mpeterson You can try out https://github.com/posquit0/Awesome-CV/pull/48

**Comment by jasonhemann** [comment]
I continue to have this problem as well. I can provide an mwe if need be.

**Comment by johannesbottcher** [comment]
@jasonhemann  Please do so.

**Comment by Tiliavir** [comment]
```% Define a subentry of cv information
% Usage: \cvsubentry{<position>}{<title>}{<date>}{<description>}
\newcommand*{\cvsubentry}[4]{%
  \setlength\tabcolsep{0pt}
  \setlength{\extrarowheight}{0pt}
  \begin{tabular*}{\textwidth}{@{\extracolsep{\fill}} L{\textwidth - 4.5cm} R{4.5cm}}
    \setlength\leftskip{0.2cm}
    \subentrytitlestyle{#2} & \ifthenelse{\equal{#1}{}}
      {\subentrydatestyle{#3}}{}
    \ifthenelse{\equal{#1}{}}
      {}
      {\subentrypositionstyle{#1} & \subentrydatestyle{#3} \\}
    \ifthenelse{\equal{#4}{}}
      {}
      {\multicolumn{2}{L{17.0cm}}{\subdescriptionstyle{#4}} \\}
  \end{tabular*}
}```

there is at least one issue with a missing `\\` in the first row of the tabular ...

**Comment by GiorgioBalestrieri** [comment]
I also have the same issue. This should be re-opened.

**Comment by gamerkhang** [comment]
I can't add subentries with nested list like what @mpeterson has, how can I fix this? The changes from @johannesbottcher weren't indented the way I wanted them to be and there was a lot of whitespace between the title and the description.

**Comment by cbtexan04** [comment]
@GiorgioBalestrieri  @gamerkhang did you guys ever find a workaround?

**Comment by gamerkhang** [comment]
nope, i only commented a couple days ago lol

**Comment by urbanhusky** [comment]
`cvsubentry` isn't working for me either. Has this been fixed?

**Comment by Berkmann18** [comment]
Same here.

**Comment by bindas1** [comment]
For anyone still looking for the answer, this has solved my issue: [#297](https://github.com/posquit0/Awesome-CV/issues/297)
